static/app/views/starfish/views/spans/selectors/actionSelector.tsx

```diff
@@ -59,6 +55,14 @@ export function ActionSelector({
               label: datum[SPAN_ACTION],
             };
           }),
+        {
+          value: EMPTY_OPTION_VALUE,
+          label: (
+            <EmptyContainer>
+              {t('(No %s)', LABEL_FOR_MODULE_NAME[moduleName])}
```

Comment:
Wonder if this should be `No Detected %s`. So we would get something like `No Detected SQL command`. That way it also can tell users to write in if they feel like something is not being detected correctly.

Reply:
I think it's great as-is, but this is a neat idea!

Reply:
im marking as a todo in the design review doc so we can discuss with team later, will keep this for now!